trello-web/src/containers/BoardList/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import SingleCard from "../../components/SingleCard";
+import {
+    Navbar,
+    NavbarBrand,
+    Nav,
+    NavItem,
+    NavLink,
+} from 'reactstrap';
+import { Container, Row, Card, CardBody, CardTitle, Button, Col } from 'reactstrap';
+
+interface BoardListProps {
+    id: number;
+    addBoard: (roomName: string) => void;
```

Comment:
Czy ten props istnieje? Narazie bym to wywalil bo i tak to bedzie polaczone z api